test/elixir_analyzer/test_suite/two_fer_test.exs

```diff
@@ -209,19 +209,4 @@ defmodule ElixirAnalyzer.TestSuite.TwoFerTest do
       end
     end
   end
-
-  describe "FunctionClauseError" do
-    test_exercise_analysis "doesn't allow raising FunctionClauseError explicitly",
-      comments_include: [Constants.solution_raise_fn_clause_error()] do
-      defmodule TwoFer do
-        def two_fer(name \\ "you") do
-          if is_bitstring(name) do
-            "One for #{name}, one for me."
-          else
-            raise FunctionClauseError
-          end
-        end
-      end
-    end
-  end
```

Comment:
It's a good choice to remove the exercise-specific test, but we need to have some sort of unit test for this check. Could you add something like in this PR https://github.com/exercism/elixir-analyzer/pull/153/files (`test/elixir_analyzer/exercise_test/common_checks/last_line_assignment_test.exs`)?